lib/blobby/gcs_store.rb

```diff
@@ -47,14 +47,14 @@ def initialize(bucket, key)
       # https://googleapis.dev/ruby/google-cloud-storage/latest/Google/Cloud/Storage/File.html#exists%3F-instance_method
       # bucket.file(key) returns nil, if the file does not exist!
       def exists?
-        gcs_file.nil? ? false : gcs_file.exists?
+        !!gcs_file&.exists?
       end
 
       # https://googleapis.dev/ruby/google-cloud-storage/latest/Google/Cloud/Storage/File.html#download-instance_method
       def read
         return nil unless exists?
 
-        content = gcs_file.download
+        content = gcs_file.download.read
```

Comment:
[IO#Read](https://ruby-doc.org/core-2.7.0/IO.html#method-c-read) 

> Opens the file, optionally seeks to the given offset, then returns length bytes (defaulting to the rest of the file). read ensures the file is closed before returning.

The filesystem store is implemented like:

```
      def read
        @path.open("rb") do |io|
          if block_given?
            while (chunk = io.read(512))
              yield chunk
            end
            nil
          else
            io.read
          end
        end
      rescue Errno::ENOENT
        nil
      end
```

Which I'm way more comfortable with. So lets do that.